src/content/blam/tags/placeholder/readme.md

```diff
@@ -1,6 +1,12 @@
 ---
 title: placeholder
 template: tag
-stub: true
+img: placeholder.jpg
+imgCaption: The object placement marker in [Sapien][] is a placeholder.
+keywords:
+  - sapien
+  - place
 ---
-...
+The placeholder tag is a simple [object][] exclusively used by the [Sapien][] editor for interactive markers (not to be confused with [model markers][gbxmodel#markers]). Placeholders can reference a [gbxmodel][] for their appearance.
```

Comment:
I'd be against that because it's going to generate false positives (e.g. "unit of measurement" linking to the "unit" tag), and it's going to be too many links. Wikipedia only tends to link things the first time they're mentioned or when convenient.